### DISCUSSION TITLE
I can't open my map file

### DISCUSSION THREAD

**Discussion by asywish** [Ceddini/CoreKeeperMapTool#20] [Feature Requests]
This software is really useful, but the bad thing is that my map file cannot be opened. I guess it is because my map is too big, it is 2000kb in size, so the web page is always loaded. I hope this bug can be fixed, thanks!

![1d8188c60531eecc761396ac29a95a3](https://user-images.githubusercontent.com/113260369/205460715-e94b527d-1f77-4e15-afd1-1f5d11c56d88.png)

**Comment by Ceddini** [comment]
Huh. That definitely shouldn't be happening. could you send me your map file? I have no access to one of that size and there might be more stuff that is up with it.

You can either send it to me on discord (Ceddini # 0001) or via email at mail@cedric-schmitt.de

**Comment by Ceddini** [comment]
I received a mapparts file with ~4MB and it's working without issue. Might be a different issue, so please send me errors from the dev console (F12).
